CONTRIBUTING.md

```diff
@@ -51,10 +50,10 @@ You may be able to find others to collaborate with for mutual benefit.
 
 ## Create the working (topic) branch
 
-Create a "topic" branch on which you will work.  The convention is to name the branch using the JIRA issue key.  If there is not already a JIRA issue covering the work you want to do, create one.  Assuming you will be working from the master branch and working on the JIRA issue `UP-123`:
+Create a "topic" branch on which you will work.  The convention is to name the branch using the GitHub issue number.  If there is not already a GitHub issue covering the work you want to do, create one.  Assuming you will be working from the master branch and working on the GitHub issue `#42`:
 
 ```shell
-git checkout -b UP-123 master
+git checkout -b GH-42 master
```

Comment:
I'm not too fond of the `GH` prefixed issue number for branch name.
I'll start a thread on the list soon to suggest moving toward [conventional commits](https://conventionalcommits.org/).